VALUES.md

```diff
@@ -73,9 +74,9 @@ Patch for temporary updates is returned via $VALUES_JSON_PATCH_PATH file and rem
 
 # Merged values
 
-A merged set of values is generated by Addon-operator when hooks or `enabled` scripts are executed or when helm charts are installed. In order to do that the global values are combined with the module values from the values.yaml and ConfigMap/addon-operator, and then the patches for the temporary update are applied.
+When hook or `enabled` script is executed or helm chart is installed, Addon-operator generates a merged set of values. This merged set combines global values from values.yaml files and ConfigMap/addon-operator and module values from the values.yaml files and ConfigMap/addon-operator. Also patches for the temporary update are applied.
```

Comment:
When the hook or `enabled` script should be executed, or helm chart is going to be installed, Addon-operator generates a merged set of values. This merged set combines:
* global values from values.yaml files and ConfigMap/addon-operator;
* module values from the values.yaml files and ConfigMap/addon-operator;
* patches for the temporary update are applied.